lib/generator.js

```diff
@@ -158,6 +159,8 @@ class Generator {
       throw new Error('Parameter entrypoint is required when using output = "string"');
     }
 
+    if (this.debug) log.setLevel('debug');
```

Comment:
Nice idea! Could we pass debug flag into template, so in template we also could write some debug logs. (Ofc this suggestion is for separate issue)

Reply:
you for sure have access to it in hooks out of the box, but filters and Nunjucks, that would be awesome, please write an issue for it